docs/developers/docker-node-key.md

```diff
@@ -13,7 +13,10 @@ First, familiarize yourself with the [guide](../nodes/docker-images.mdx) on how
 Run the Docker image (in this example, we are using a Light Node):
 
 ```bash
-docker run -e NODE_TYPE=light -e P2P_NETWORK=mocha -p 26659:26659 celestia-node celestia light start --core.ip https://rpc-mocha.pops.one --gateway --gateway.addr 127.0.0.1 --gateway.port 26659 --p2p.network mocha
+docker run --name celestia-node -e NODE_TYPE=light -e P2P_NETWORK=mocha -p 26659:26659 \
+ghcr.io/celestiaorg/celestia-node:sha-747c9e5 celestia light start \
```

Comment:
I would avoid sha tags in documentation as they might get pruned from the registry, and then the command is broken. I propose using the latest stable tag e.g. `0.6.3`.

Reply:
yes, agreed. the last version wasn't cut with a tag, which is why it's a SHA. DMd you relevant thread in Slack

Reply:
I'll add this as a variable in the future, similar to in this PR: https://github.com/celestiaorg/docs/pull/448

Reply:
So you plan to update the doc each time there is a new stable version?

Reply:
we'll run a workflow to update variables, per network, which will be set across docs accordingly. this will streamline updating versions on docs

is there anything that can be done to make sure future releases of docker images are released with tags as they were previously?

Reply:
also, made an issue for tracking purposes so we can have this set up for next releases: https://github.com/celestiaorg/celestia-node/issues/1702